model-metadata/NIH-RSV_WIN.yaml

```
@@ -3,6 +3,11 @@ team_abbr: "NIH"
 model_name: "RSV_WIN"
 model_abbr: "RSV_WIN"
 model_contributors: [
+   {
+   "name": "Chelsea Hansen",
+    "affiliation": "Fogarty International Center, National Institutes of Health",
+    "email": "chelsea.hansen@nih.gov"
+  }
```

```suggestion
  },
```

The metadata file is missing a `,` after your name, affiliation, etc. information. Would it be possible to add it, please?